light-node/Cargo.toml

```diff
@@ -16,6 +16,7 @@ jsonrpc-http-server = "14.2"
 jsonrpc-derive = "14.2"
 serde = { version = "1", features = ["serde_derive"] }
 serde_json = "1.0"
+sled = "0.31.0"
```

Comment:
To use the ` tendermint_light_client::store::sled::SledStore` you need to pass in a `SledDb`. This could be abstracted away in that library s.t. users don't need the extra sled dependency. Wanted to keep changes as contained in the light-node crate as much as possible. Maybe a good candidate for a followup PR?

Reply:
I think it's fine, if we decide to make `sled` a public dependency of our API then the caller ought to construct the db. A potential change would be to hide away sled entirely, which I think is not the worst idea, and rework it from `SledDb` to `OnDisk` or similar.

Reply:
Linked in #428